00_Base/package.json

```diff
@@ -32,8 +32,8 @@
     "@types/lodash.startcase": "^4.4.9"
   },
   "dependencies": {
-    "@citrineos/base": "1.3.1",
-    "@citrineos/data": "1.3.1",
+    "@citrineos/base": "1.3.2",
+    "@citrineos/data": "1.3.2",
```

Comment:
won't this still break, since 1.3.2 is not on npm yet?

Reply:
I adjusted our workflow to git clone a specific branch because otherwise it fails the other way where it pulls `ocpi` main which doesnt have new code for example the RealTimeAuthorizer, and then fails